unit_tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Unit Testing Netguard
+
+## Requirements
+An Android x86/64 emulator with no Google Play
+
+## Running Tests
+* Build the app or just the `vpn-network-imp` module
+* Execute the following:
+```
+cd android-app/vpn-network/vpn-network-impl/build/intermediates/cmake/debug/obj
+adb push x86_64/ /data/
```

Comment:
This step fails on emulator with `permission denied`. Are we sure we don't need a rooted device?